src/cpp/session/SessionSourceDatabaseSupervisor.cpp

```diff
@@ -354,18 +351,29 @@ bool reclaimOrphanedSession(FilePath* pSessionDir)
 
    BOOST_FOREACH(const FilePath& sessionDir, sessionDirs)
    {
+      // if the suspend file exists, this session is only sleeping, not dead
+      if (sessionSuspendFilePath(sessionDir).exists())
```

Comment:
These files have effectively unlimited age as sessions can be suspended indefinitely in server mode.